practice-two/src/styles/bases/base.scss

```diff
@@ -5,4 +5,5 @@
 html {
   font-family: Integral CF, Satoshi;
   font-size: $size;
+  max-width: 1440px;
```

Comment:
Why do we need max-width? What would the UI look like if it were larger than 1440px? Will there be white space on the left and right sides?